tests/helix/send-to-helix-basictests.targets

```diff
@@ -32,7 +32,7 @@
       <!-- runsettings timeout in ms, default to 15 mins -->
       <_DefaultWorkItems TimeoutMs="900000" />
 
-      <_DefaultWorkItems Condition="'%(FileName)' == 'Aspire.Hosting.Elasticsearch.Tests'" TimeoutMs="1200000" />
+      <_DefaultWorkItems Condition="'%(FileName)' == 'Aspire.Hosting.Elasticsearch.Tests'" TimeoutMs="3600000" />
```

Comment:
Is this really an hour? We are going to wait an hour for these tests? Isn't that just going to make us wait longer when the thing isn't responsive?

Reply:
This will become a problem whenever helix is backed up. The total timeout for the whole job is 90mins, so this will definitely timeout. Do we need to break this up?
https://github.com/dotnet/aspire/blob/2344aaf77fadfecd3186a42205882bea56885b8a/eng/pipelines/azure-pipelines-public.yml#L102-L103

Reply:
The test run on this PR took ~8mins. If this is taking up to an hour sometimes then maybe we need to investigate why that is the case.